packages/ui/src/council/modals/AnnounceCandidacy/components/TitleAndBulletPointsStep.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import * as Yup from 'yup'
+
+import { InputComponent, InputText } from '@/common/components/forms'
+import { Row } from '@/common/components/Modal'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { useForm } from '@/common/hooks/useForm'
```

Comment:
I think that we could re-visit `useForm()` here. It looks like it is not used in 100%